frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -510,6 +498,35 @@ The interval value is in seconds and can range from 3 to 20 seconds.]]),
             end,
         })
     end
+    table.insert(menu_items.long_press.sub_item_table, {
+        text_func = function()
+            return T(_("Highlight long-press interval: %1 s"),
+                G_reader_settings:readSetting("highlight_long_hold_threshold_s", 3))
+        end,
+        keep_menu_open = true,
+        callback = function(touchmenu_instance)
+            local SpinWidget = require("ui/widget/spinwidget")
+            local items = SpinWidget:new{
+                title_text = _("Highlight long-press interval"),
+                info_text = _("If a touch is not released in this interval, it is considered a long-press. On document text, single word selection will not be triggered."),
+                width = math.floor(Screen:getWidth() * 0.75),
+                value = G_reader_settings:readSetting("highlight_long_hold_threshold_s", 3),
+                value_min = 2.5,
+                value_max = 20,
+                value_step = 0.5,
+                value_hold_step = 1,
```

Comment:
I thought we settled on this? Might be mistaken. :-P
```suggestion
                value_step = 0.1,
                value_hold_step = 0.5,
```